src/components/Welcome.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import welcomeAction from '../redux/action-creators';
+
+class Welcome extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  componentDidMount() {
+    console.log(this.props);
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <h1 className="app-title">Authors Haven</h1>
+        <p>Welcome</p>
+      </div>
+    );
+  }
+}
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
Another way of doing this could be: 

```javascript 
const actionCreators = {
welcomeAction
}
```

Or in the bottom, 
```javascript
export default connect(
  mapStateToProps, {welcomeAction}
)(Welcome);
```
This second is efficient where action creator is one, but the first choice could be better for where action creators is more than one.

Reply:
Did you try any of these?